### ISSUE TITLE
ob_get_status buffer_size vs chunk_size

### ISSUE THREAD

**Issue by kkmuffme** [php/doc-en#3816]
https://www.php.net/manual/en/function.ob-get-status.php

Please improve the documentation of `buffer_size` - it's the currently allocated maximum memory for this output buffer. It's at least 4096 bytes (1 memory page, could be 8k on some systems) and dynamically resized depending on size of the output and always a multiple of 4096 (or 8192 I guess if memory page is 8k)

Also chunk_size of 0/1 seems to have a special value for `buffer_size` - see https://github.com/php/php-src/issues/16135, should probably be documented on https://www.php.net/manual/en/function.ob-start.php

**Comment by cmb69** [comment]
I agree that the documentation needs to be improved. See https://github.com/php/php-src/pull/16161#issuecomment-2391287052, for some comments which should better have been posted here in the first place.
